File: Python/somaVetor.py
N: int
soma: float
media: float

N = int(input("Quantos números você vai digitar? "))

vet: [float] = [0 for x in range(N)]

soma = 0
for i in range(0, N):
    vet[i] = float(input("Digite um número: "))
    soma = soma + vet[i]

media = soma / N

print(f"\nVALORES = ", end="")
for i in range(0, N):
    print(f"{vet[i]:.1f} ", end="")

print(f"\nSOMA = {soma:.2f}")
print(f"MEDIA = {media:.2f}")

File: C/soma_vetor.c
#include<stdio.h>

int main()
{

    int i, N;
    double soma, media;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &N);

    double vet[N];

    for(i=0; i < N; i++)
    {
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
    }

    soma = 0;
    for(i=0; i < N; i++)
    {
        soma = vet[i] + soma;
    }

    media = soma / N;

    printf("\nVALORES = "); // vetor
    for(i=0; i < N; i++)
    {
        printf("%.1lf ", vet[i]);
    }

    printf("\nSOMA = %.2lf", soma); // soma
    printf("\nMEDIA = %.2lf", media); // media

    return 0;
}
